src/migrateCalculatedMeasuresToActivePivot/migrateCalculatedMeasuresInWidgets.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  ContentRecord,
+  CubeName,
+  DataModel,
+  MdxSelect,
+  WidgetWithQueryState,
+} from "@activeviam/activeui-sdk";
+import { MdxString } from "@activeviam/mdx";
+import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
+import { produce } from "immer";
+import _mapValues from "lodash/mapValues";
+
+export const migrateCalculatedMeasuresInWidgets = (
```

Comment:
Let's explain roughly what we're doing:
```suggestion
/**
* In ActiveUI 5.0, saved calculated measures live under /ui/calculated_measures, and ActivePivot is not aware of them. In particular, they don't appear in the data model, and they must be added as query-scoped measures to each individual widget refering to them.
* 
* In ActiveUI 5.1, saved calculated measures are grouped by cube and live under /pivot/entitlements/cm. ActivePivot proxies them and makes them accessible in the data model. So they can be referred to just like any other measure in an MDX query, without the need to add a query-scoped calculated measure definition.
*/
export const migrateCalculatedMeasuresInWidgets = (
```